addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/onboard-account.cfn.yml

```diff
@@ -287,6 +287,11 @@ Resources:
               - ssm:GetParameter
             Resource:
               - !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/*'
+          - Effect: Allow
+            Action:
+              - cloudformation:DescribeStacks
+              - cloudformation:GetTemplate
+            Resource: !Sub 'arn:aws:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/SWB-*'
```

Comment:
Is this naming convention backward compatible with existing onboard stacks?

Reply:
Not yet, I haven't worked out what the naming convention will be, I'm pretty sure that the existing stacks are prefixed with 'swb-', but I'm not completely sure. This part was mainly a placeholder for testing purposes for now, it's on my list of things to fix prior to merge to develop but I can take a look at it as part of this PR too if you'd rather it not wait. I need to take a deeper look at the naming of the CFN stacks as part of implementing the onboarding process itself.